src/_zkapauthorizer/_storage_server.py

```diff
@@ -360,6 +366,7 @@ def _slot_testv_and_readv_and_writev(
             if required_new_passes > len(validation.valid):
                 validation.raise_for(required_new_passes)
 
+        self._original.set_implicit_slot_lease_renewal(renew_leases)
```

Comment:
I was curious why this switched to being a instance attribute on the wrapped server, but I see you change to set this globally in #249. As mentioned in the tahoe PR this depends on, I'm not sure that having an instance attribute, as opposed to a parameter is the best interface, though.